### ISSUE TITLE
Geojson data problem generated by api.to_geojson function

### ISSUE THREAD

**Issue by geoslegend** [sentinelsat/sentinelsat#110]
Hi all,

I test an example code on Python 2.7.10 (Windows 10). The code generates the geojson file using api.to_geojson to preview the boundaries of sentinel-x data. However, the geojson file could NOT be loaded in QGIS 2.18.2.

After comparing of the geojson file with problems and the geojson data generated from geojson.io (http://geojson.io) website, I could find some differences as follows:

by api.to_geojson
```
     "coordinates": [
        [107.734726, 32.666718],
        [110.418823, 33.071476],
        [110.739563, 31.448975],
        [108.104744, 31.041954],
        [107.734726, 32.666718]
      ],
```

by geojson.io
```
     "coordinates": [[
        [107.734726, 32.666718],
        [110.418823, 33.071476],
        [110.739563, 31.448975],
        [108.104744, 31.041954],
        [107.734726, 32.666718]
      ]],
```

I could load the geojson file without any problems in QGIS 2.18.2 after adding extra brackets to 'coordinates' attributes using notepad program.

I think that the **to_geojson** function should be improved to solve the problem of geojson data generated by the function.

**Comment by Fernerkundung** [comment]
You are right, our current implementation violates the [GeoJSON spec on polygons](https://tools.ietf.org/html/rfc7946#section-3.1.6) where the coordinates member shall be an "an array of linear ring coordinate arrays". As you pointed out - this needs to be nested more deeply.

**Comment by Fernerkundung** [comment]
Fixed in `v0.10`.
